src/content/docs/de/basics/astro-components.mdx

```diff
@@ -251,10 +251,10 @@ import Wrapper from '../components/Wrapper.astro';
 Verwende ein `slot="mein-slot"`-Attribut auf dem untergeordneten Element, das du an einen passenden `<slot name="mein-slot" />`-Platzhalter in deiner Komponente weiterleiten willst.
 :::
 
-Um mehrere HTML-Elemente ohne ein umschließendes `<div>` in den `<slot/>`-Platzhalter einer Komponente zu übergeben, verwende das `slot=""`-Attribut auf [Astros <Fragment/>-Komponente](/de/reference/api-reference/):
+Um mehrere HTML-Elemente ohne ein umschließendes `<div>` in den `<slot/>`-Platzhalter einer Komponente zu übergeben, verwende das `slot=""`-Attribut auf [Astros <Fragment/>-Komponente](/de/reference/components-reference/):
 
 
-```astro title="src/components/CustomTable.astro" "<slot name="header"/>" "<slot name="body"/>"
+```astro title="src/components/CustomTable.astro" '<slot name="header" />' '<slot name="body" />'
```

Comment:
Not sure how to handle this one here, since they are currently not being highlighted in the English version.

Reply:
I would just highlight them cause I think it's intended to be highlighted.